user/enterprise/high-availability.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Enterprise High Availability Mode
+layout: en_enterprise
+
+---
+
+Travis CI Enterprise typically runs as a single container instance communicating with one or multiple workers and generally, this setup is quite stable and maintainable. However, in some cases -- especially volume licensing or by corporate requirement -- you might need to install Travis CI in High Availability (HA) mode. 
+
+If you're interested, or might be interested, in HA, please let [us](mailto:enterprise@travis-cicom?subject:HA%20Mode) know and we can discuss options and help you get started. 
```

Comment:
I'd spell out HA here, and probably in general? :man_shrugging: unless it is an important industry term?

Reply:
Something slightly off about the grammar around these two options:

> However, in some cases -- especially volume licensing or by corporate requirement -- you might need to install Travis CI in High Availability (HA) mode.

maybe especially *for* volume licensing?

Reply:
I'd leave it as "If you're interested" and let people figure out if they might be or not :-p

Reply:
It is a standard term, but I can spell it out more!

Reply:
I changed the first paragraph entirely -- it seemed a bit unfocused.